docs/tutorials/proposals/new-market-proposal.md

```diff
@@ -67,9 +67,9 @@ Decimal places need to be defined for both order sizes and the market.
 * `decimalPlaces` - sets the smallest price increment on the book
 * `positionDecimalPlaces` - sets how big the smallest order / position on the market can be
 
-Timestamps are required for ending the voting period, as well as enacting the changes.
+Timestamps are required for ending the voting period, as well as enacting the market. The time between closing and enactment also defines how long an opening auction will be. It cannot be longer than 7 days.
```

Comment:
Is it 7 days or some network parameter that is initially 7 days

Reply:
It’s the difference between two network parameters, maxClose and maxEnact